pandas/tests/test_window.py

```diff
@@ -339,6 +344,13 @@ def test_constructor(self):
                     c(window=2, min_periods=w)
                 with self.assertRaises(ValueError):
                     c(window=2, min_periods=1, center=w)
+        # GH 13383
```

Comment:
make this a separate test (as skipping will cause the entire rest of the test to be skipped)